Maths/test/CoPrimeCheck.test.js

```diff
@@ -0,0 +1,29 @@
+import { CoPrimeCheck } from '../CoPrimeCheck'
+
+describe('CoPrimeCheck', () => {
+  it.each([
+    [1, 1],
+    [1, 2],
+    [1, 3],
+    [1, 7],
+    [20, 21],
+    [5, 7],
+    [-5, -7]
+  ])('returns true for %j and %i', (inputA, inputB) => {
+    expect(CoPrimeCheck(inputA, inputB)).toBe(true)
+    //expect(CoPrimeCheck(inputB, inputA)).toBe(true)
```

Comment:
Any reason for this to be commented out? Having the symmetric test seems good to me.

Reply:
No, there is no reason. I was just doing some _debugging_ and forget to uncomment it. Thanks for spotting that.